src/java/com/wolfssl/WolfSSL.java

```diff
@@ -880,6 +895,24 @@ public static native int getPkcs8TraditionalOffset(byte[] in, long idx,
         return suiteArray;
     }
 
+    /**
+     * Gets a list of all cipher suites available for current native wolfSSL
+     * configuration and selected protocol level. In the format TLS_*.
+     *
+     * @param version protocol version for which to get cipher suites.
+     * @return list of cipher suites.
+     */
+    public static String[] getCiphersAvailableIana(TLS_VERSION version) {
+        /* passing Enum as ordinal to JNI layer, see com_wolfssl_WolfSSL.c */
+        String cipherSuites = getAvailableCipherSuitesIana(version.ordinal());
+        if (cipherSuites == null)
+            return null;
+
+        String[] suiteArray = cipherSuites.split(":");
```

Comment:
this works but is a couple extra steps, lets return cipherSuties.split(":") directly

Reply:
Great feedback, fixed this issue and multi-threaded one above.